src/Sql/Sql.Test/Sql.Test.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="Microsoft.Azure.KeyVault.WebKey" Version="3.0.1" />
     <PackageReference Include="Microsoft.Azure.Management.KeyVault" Version="2.4.2" />
     <PackageReference Include="Microsoft.Azure.Management.OperationalInsights" Version="0.19.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.37.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.38.0-preview" />
```

Comment:
Azure sdk generation is currently blocked so 1.38.0 build doesn't actually exist.  However, I set it here so that once that sdk is generated, no changes have to be made here.